res/static/fragment_callback.js

```diff
@@ -0,0 +1,14 @@
+document.addEventListener('DOMContentLoaded', function() {
+  var form = document.getElementById('form');
+  var params = location.hash.slice(1);
+  var re = /([^&=]+)=([^&]*)/g;
```

Comment:
I'm not very happy we have to use a custom regex here. But I also have no better solution.